controllers/dspipeline_controller.go

```diff
@@ -255,7 +264,7 @@ func (r *DSPAReconciler) Reconcile(ctx context.Context, req ctrl.Request) (ctrl.
 		return ctrl.Result{}, err
 	}
 
-	conditions, err := r.GenerateStatus(ctx, dspa)
+	conditions, err := r.GenerateStatus(ctx, dspa, params)
```

Comment:
When deploying the DSPA for the first time, I don't see Statuses show up until the default mariadb/minio are deployed. This is problematic because ODH Dashboard relies on the "Ready" condition type to assess the current status of the DSPA.